src/components/CreateList.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  <div @click="closeForm" class="fullscreen">
+    <form v-on:submit.prevent="">
```

Comment:
Please use common style to define event handler in whole project @ or v-on but not both